### DISCUSSION TITLE
NVIM + Copilot  466 Please upgrade your Copilot extension to continue using this service

### DISCUSSION THREAD

**Discussion by draganjovanovich** [community/community#15684] [Copilot Conversations] [answered]
Hi,

NVIM v0.6.1
[Copilot.vim 1.2.2](https://github.com/github/copilot.vim/commit/573da1aaadd7402c3ab22fb1ae6853db1dc82acb)

Using this nvim plugin.

I have Copilot working normally until couple days ago, and now i get this message in logs:

`<-! [ERROR] [streamChoices] [2022-04-27T15:30:10.314Z] Unhandled status from server: 466 Please upgrade your Copilot extension to continue using this service.`

There is no update available for vim.copilot.

I saw that you had a problem on server side some 13 days ago, but i have this problem currently  and i am not sure is it backend related ?

Thanks,

Dragan.

**Comment by hsenag** [comment, marked as answer]
Hi, I've just double-checked and the latest version is working fine. Please could you check if you are actually running the version you thought - might there be an old version lying around somewhere that's getting picked up instead?

**Reply by atsepkov** [reply]
I just started running into the same issue, it worked last week and today I see the same error in my logs.

**Reply by hsenag** [reply]
> I just started running into the same issue, it worked last week and today I see the same error in my logs.

Please double-check that you are using the latest version.

**Reply by atsepkov** [reply]
Ok, it was indeed a version update, `PlugUpdate` seems to be failing to pick up the update, navigating to the install directory and running `git pull` manually fixed the issue.

**Reply by Aaronarphaxad** [reply]
I am currently having the same issue. 

`[INFO] [default] [2022-05-18T18:11:37.227Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-05-18T18:11:37.677Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 449 ms
[INFO] [fetch] [2022-05-18T18:11:37.680Z] Please upgrade your Copilot extension to continue using this service.`

**Reply by Kalebu** [reply]
I think this is due to change in authentication between Github and Vscode 

I was having the same issue but was quickly solved by updating the Vscode to the latest version

**Reply by ghost** [reply]
I've seen this on vscode version `1.66.1` (from March 2022) and I can confirm it's resolved in `1.67.2` from April.

**Comment by Aaronarphaxad** [comment]
I will try that. Thank you

**Reply by sanhuafeiluo** [reply]
升级vscode后，问题已解决。
After upgrading vscode, the problem has been solved.

**Reply by ykf904839896** [reply]
yep solved

**Comment by nicobala89** [comment]
I've got the same error:
[fetch] [2022-06-14T04:40:14.077Z] Please upgrade your Copilot extension to continue using this service.
I use vscode in a WSL2 and I all ready upgrade my linux distro, the vscode,  even I reinstall vscode, but the issue still occurrs...
Any suggestions?

**Reply by hsenag** [reply]
What versions of VSCode and Copilot do you have?

**Reply by nicobala89** [reply]
For Copilot v1.29.6152 and VSCode v1.68.0 (user setup)

**Reply by hsenag** [reply]
You definitely shouldn't be getting that error with that version. Can you double-check that's what's installed in the WSL2 container?

**Reply by nicobala89** [reply]
Yes I double-check, but I fixed the error unIstalling VSCode  and removing manually all residual files related with VSC in windows and Linux (WSL2), and then installing again VSC and it's working, maybe some corrupt file was interfiring with copilot

**Comment by transcendr** [comment]
For those using packer, try:
1. uninstall and reinstall copilot
2. `:Copilot restart` or restarting nvim if still not working

As someone suggested before, you can try manually running `git pull` in the plugin directory:
`cd ~/.local/share/nvim/site/pack/packer/opt/copilot.nvim`

**Comment by d33tah** [comment]
I had a similar problem. I updated the plugin and it persisted, so I went to `~/.config/nvim/pack/github/start/copilot.vim/`, ran `git reset --hard && git pull` and it fixed it for me.
